cmd/node/config/p2p.toml

```diff
@@ -24,6 +24,10 @@
     #the sync and consensus mechanisms
     ThresholdMinConnectedPeers = 3
 
+    # WaitNumPeersOnBootstrap is the minimum number of peers to wait on bootstrap or the node will wait the default
+    # time which is now set to ~20 seconds
+    WaitNumPeersOnBootstrap = 10
```

Comment:
Perhaps rename it to  smth like:`MinNumPeersToWaitForOnBootstrap`?

Also, maybe it would be worth to add in the description what is the name of the variable which defines:
> the default time which is now set to ~20 seconds

since its value might be changed

Reply:
done